src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,67 @@
+package mate.academy.model;
+
+import com.sun.istack.NotNull;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+    @Column(length = 6555)
```

Comment:
'unique = true' should also be declared here
![image](https://user-images.githubusercontent.com/117547096/206291154-083a3ada-e04a-4701-9b45-215429521114.png)